src/controllers/router-store/index.tsx

```diff
@@ -157,6 +157,7 @@ const actions: AllRouterActions = {
               ...nextContext,
               location,
               action,
+              state: location.state,
```

Comment:
Given we are already storing `location`, does it make sense to store `state` here too? Wouldn't it be duplicated?

Reply:
Good point. I can remove this line. I can either leave it inside the location or abstract it out and so it is easier to access.